packages/ui/react-ui-editor/src/styles/markdown.ts

```diff
@@ -6,13 +6,14 @@ import { mx } from '@dxos/react-ui-theme';
 
 export type HeadingLevel = 1 | 2 | 3 | 4 | 5 | 6;
 
+// https://tailwindcss.com/docs/font-weight
 export const heading: Record<HeadingLevel, string> = {
-  1: 'mbs-4 mbe-2 text-4xl font-semibold text-inherit no-underline',
-  2: 'mbs-4 mbe-2 text-3xl font-bold text-inherit no-underline',
-  3: 'mbs-4 mbe-2 text-2xl font-bold text-inherit no-underline',
-  4: 'mbs-4 mbe-2 text-xl font-extrabold text-inherit no-underline',
-  5: 'mbs-4 mbe-2 text-lg font-extrabold text-inherit no-underline',
-  6: 'mbs-4 mbe-2 font-black text-inherit no-underline',
+  1: 'mbs-4 mbe-2 text-4xl font-body font-medium text-inherit no-underline',
+  2: 'mbs-4 mbe-2 text-3xl font-body font-medium text-inherit no-underline',
+  3: 'mbs-4 mbe-2 text-2xl font-body font-medium text-inherit no-underline',
+  4: 'mbs-4 mbe-2 text-xl font-body font-medium text-inherit no-underline',
+  5: 'mbs-4 mbe-2 text-lg font-body font-medium text-inherit no-underline',
+  6: 'mbs-4 mbe-2 font-body font-medium text-inherit no-underline',
```

Comment:
`font-body` should be inherited from the markdown language style, is that not the case? Otherwise this sets the precedent that _all highlightings_ for markdown should set the font family, which should be unnecessary.